test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/s2i/KieServerS2iHierarchicalIntegrationTest.java

```diff
@@ -143,10 +144,10 @@ public void testContainerAfterExecServerS2iStart() {
         Long processId = processServicesClient.startProcess(CONTAINER_ID, Constants.ProcessId.USERTASK);
         assertThat(processId).isNotNull();
 
-        List<TaskSummary> tasks = taskServicesClient.findTasks(Constants.User.YODA, 0, 10);
+        List<TaskSummary> tasks = taskServicesClient.findTasks(DeploymentConstants.getAppUser(), 0, 10);
```

Comment:
I think we can keep constants for task service, considering that the user in bpmn file is hardcoded too.

Reply:
I would prefer to have the less hardcoded values the better. Right now, the only adminUser is in the bpmn files and I would like to provide a solution to remove from here as well (but as part of another ticket). What do you think about?

Reply:
Sounds ok.